app/views/users/_form.html.erb

```diff
@@ -10,69 +10,117 @@
       </ul>
     </div>
   <% end %>
+  
+  <div class="container">
+    <div class="py-5 text-center">
+      <h2>Registration</h2>
+    </div>
 
-<%# ............................temp......................... %>
-  <section class="vh-100">
-  <div class="container-fluid h-custom">
-    <div class="row d-flex justify-content-start align-items-center h-100">
-      <div class="col-md-9 col-lg-6 col-xl-5">
-        <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/draw2.webp" class="img-fluid"
-          alt="Sample image">
-      </div>
-      <div class="col-md-10 col-lg-8 col-xl-6 offset-xl-1">
-        <form>
-          <div class="divider d-flex align-items-center my-4">
-            <p class="text-center fw-bold mx-3 mb-0">SIGNUP</p>
-          </div>
-
-          
-          <div class="align-items-center">
-            <!-- Name input -->
-            <div class="col-md-7 form-outline mb-4">
-              <%= form.text_field :name, class: "form-control form-control-lg", type: "name", placeholder: "Enter Your Name" %>
+    <div class="row">
+      <div class="col-md-8 order-md-1">
+        <h4 class="mb-3">SignUp Form</h4>
+        <form class="needs-validation" novalidate>
+          <div class="row">
+            <div class="col-md-6 mb-3">
+              <label for="firstName">Name*</label>
+              <%= form.text_field :name, class: "form-control", type: "name", placeholder: "" %>
+              <div class="invalid-feedback">
```

Comment:
who does `<div class="invalid-feedback">` is working?

Reply:
done